### PR TITLE
Updating the Website

### PR DESCRIPTION
**Pull request by maggiehollis**
Updating the [website](https://gradbench.org) to include relevant information and links.

### CODE REVIEW

File: package-lock.json
Comment (samestep) [inline package-lock.json:1]: What version of Node.js do you have installed? I suspect you may have an old version that is causing this crazy diff on `package-lock.json`. Let's revert the diff on this file before we merge this PR.

File: packages/gradbench/src/App.css
Comment (samestep) [inline packages/gradbench/src/App.css:6]: Maybe get rid of the margin around the header at the top of the page?

```suggestion
body {
  margin: 0;
```

File: packages/gradbench/src/App.tsx
Comment (samestep) [inline packages/gradbench/src/App.tsx:32]:
```suggestion
        are written in <b>Adroit</b> and automatically translated to a tool's
```

File: packages/gradbench/src/App.tsx
Comment (samestep) [inline packages/gradbench/src/App.tsx:10]: Can we replace this with something a bit more descriptive/catchy? Maybe use the tagline in the "about" section of the GitHub repo:

```suggestion
            a benchmark for differentiable programming across languages and domains
```

And I'd probably make it lowercase so it feels more like a description and less like a subtitle.

File: packages/gradbench/src/App.tsx
Comment (samestep) [inline packages/gradbench/src/App.tsx:33]:
```suggestion
        native language. The architecture allows for the easy addition of new
```

File: packages/gradbench/src/App.tsx
Comment (samestep) [inline packages/gradbench/src/App.tsx:42]:
```suggestion
      <p className="text">TODO: Explain Adroit</p>
```

File: packages/gradbench/src/App.tsx
Comment (samestep) [inline packages/gradbench/src/App.tsx:127]:
```suggestion
          <a href="https://jax.readthedocs.io/en/latest/index.html">JAX</a>
```

File: packages/gradbench/src/App.tsx
Comment (samestep) [inline packages/gradbench/src/App.tsx:118]: Instead of having a separate "links" section, could we have each of the individual row headers in the table above be links?

**Review by samestep — APPROVED** [review]
Amazing, this looks super cool! If you're down, we can just undraft and merge this PR (after making a couple changes mentioned below), and then continue improving on the website in followup PRs.

File: package-lock.json
Comment (maggiehollis) [reply package-lock.json:1]: I think it was a problem with my node_modules

**Pull request merged**
